modules/localStore.js

```diff
@@ -0,0 +1,24 @@
+class LocalStore {
+  static getBooks() {
+    let books;
+    const localBook = localStorage.getItem('local');
+    if (!localBook) {
+      books = [];
+    } else {
+      books = JSON.parse(localBook);
+    }
+    return books;
+  }
+
+  static addBooks(newBook) {
```

Comment:
As requested by the previous reviewer, please use arrow functions instead of normal functions 😉.
Tip: you could have something like this:

![image](https://user-images.githubusercontent.com/42799579/207378445-b0171248-3514-45a3-ad1a-1ccd0113bef6.png)

Check [this link](https://www.digitalocean.com/community/tutorials/understanding-arrow-functions-in-javascript) to learn more about arrow functions.

As you are using static functions, adapting the code I showed above may introduce you to new errors or even stop your code from working 😄. To overpass that, try this code:

![image](https://user-images.githubusercontent.com/42799579/207381605-dc2ef039-d5a2-416b-bf1c-5ec4c1fbeb13.png)

It will probably work as you expect and avoid unintended errors while using the modern syntax 😉.